app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionsListFragment.kt

```diff
@@ -175,21 +191,23 @@ class ConnectionsListFragment : BaseFragment(), ListItemClickListener, View.OnCl
         binding.executePendingBindings()
     }
 
-    private fun showPopupMenu(view: View?, item: ConnectionViewModel) {
+    //TODO REFACTOR TO STANDALONE CLASS (example https://github.com/zawadz88/MaterialPopupMenu)
+    private fun showPopupMenu(parentView: View?, anchorView: View?, item: ConnectionViewModel): PopupWindow? {
+        if (parentView == null || anchorView == null) return null
         try {
             val layoutInflater = context?.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
             val popupView = layoutInflater.inflate(R.layout.view_popup_menu, null)
-            popupWindow = PopupWindow(
+            val popupWindow = PopupWindow(
```

Comment:
why u create new val?

Reply:
to exclude optional value unwrapping during PopupWindow setup